src/datasources/GenerateCollectionVariablesAPI.js

```diff
@@ -0,0 +1,40 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+import { camelCase, mapKeys } from 'lodash'
+
+export class GenerateCollectionVariablesAPI extends RESTDataSource {
```

Comment:
Can we rename this file and class to keep consistent with the other datasources, please.

Reply:
Renamed to CollectionVariableDrafts